torch_geometric/nn/models/basic_gnn.py

```diff
@@ -57,7 +63,8 @@ def __init__(
         out_channels: Optional[int] = None,
         dropout: float = 0.0,
         act: Union[str, Callable, None] = "relu",
-        norm: Optional[torch.nn.Module] = None,
+        norm: Optional[str] = None,
```

Comment:
Should this be a `Union[str, Callable]`?

Reply:
Good catch. Fixed.